Datacite JSON.js

```diff
@@ -260,7 +331,9 @@ function doImport() {
 			}
 		}
 	}
-	
+	// remove duplicate creators (they'll have had multiple roles in datacite metadata)
+	let uniqueCreatorSet = new Set(item.creators.map(JSON.stringify));
```

Comment:
This was the best way I could find to replace duplicates in the creator array (example is from https://doi.org/10.48548/pubdata-155, the 3rd last test). Lmk if there's a better way to do this